ddl/placement_policy.go

```diff
@@ -232,6 +230,33 @@ func onAlterPlacementPolicy(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		return ver, errors.Trace(err)
 	}
 
+	ids, err := getPlacementPolicyDependedObjectsIDs(t, oldPolicy)
+	if err != nil {
+		return ver, errors.Trace(err)
+	}
+	if len(ids) != 0 {
+		// build bundle from new placement policy.
+		bundle, err := placement.NewBundleFromOptions(newPolicyInfo.PlacementSettings)
+		if err != nil {
+			return ver, errors.Trace(err)
+		}
+		err = bundle.Tidy()
+		if err != nil {
+			return ver, errors.Trace(err)
+		}
+		// Do the http request only when the rules is existed.
+		bundles := make([]*placement.Bundle, 0, len(ids))
+		for _, id := range ids {
+			cp := bundle.Clone()
+			bundles = append(bundles, cp.Reset(id))
```

Comment:
It seems we use the object id as the rule id when creating it. Is it better to make a more semantic names like `pl_tb_1` ? `pl` is short for 'placement rule' and `tb_1` stands for it is a rule group for table with id 1. It may be easier to maintain and distinguish from other rules created by others.

Reply:
However, It is not the work for current pr. May be we can discuss it in the weekly meeting.

Reply:
you means the group_id? yes it's kind of customized format, we can talk about it to make it sense